### PR TITLE
Data-Ajax: fix for when ajaxing templating logic

### PR DESCRIPTION
**Pull request by Garneauma**

### CODE REVIEW

**Review by duboisp — CHANGES_REQUESTED** [review]
Can we add a simple example that show the issue we were having?

**Comment by Garneauma** [comment]
> Can we add a simple example that show the issue we were having?

Working example added. I also changed my solution. This one fixes the issue with very little change to the code.

**Review by duboisp — CHANGES_REQUESTED** [review]

File: src/plugins/ajax-fetch/ajax-fetch.js [hunk 1]
```diff
@@ -81,7 +81,15 @@ $document.on( "ajax-fetch.wb", function( event ) {
 				fetchData.pointer = $( "<div id='" + wb.getId() + "' data-type='" + responseType + "'></div>" )
 					.append( responseType === "string" ? response : "" );
 
-				response = !xhr.responseJSON ? $( response ) : xhr.responseText;
+				if ( !xhr.responseJSON ) {
+					try {
+						response = $( response );
+					} catch ( e ) {
+						response = DOMPurify.sanitize( xhr.responseText );
```
Comment (duboisp) [inline src/plugins/ajax-fetch/ajax-fetch.js:88]: DOMPurify need to be mapped in the plugin closure mapping.

**Review by duboisp — CHANGES_REQUESTED** [review]
Tested and reviewed the code. Everything work as expected.

Just one little change, see inline comment.

**Review by duboisp — APPROVED** [review]
Change requested was completed

**Pull request merged**